tests/blazesym.rs

```diff
@@ -124,6 +126,22 @@ fn symbolize_elf_dwarf_gsym() {
     test(src, true);
 }
 
+/// Check that we "fail" symbolization as expected on a stripped ELF
+/// binary.
+#[test]
+fn symbolize_elf_stripped() {
+    let path = Path::new(&env!("CARGO_MANIFEST_DIR"))
+        .join("data")
+        .join("test-stable-addresses-stripped.bin");
+    let src = symbolize::Source::Elf(symbolize::Elf::new(path));
+    let symbolizer = Symbolizer::new();
+    let result = symbolizer
+        .symbolize_single(&src, symbolize::Input::VirtOffset(0x2000100))
+        .unwrap();
+
+    assert_eq!(result, Symbolized::Unknown(Reason::MissingSyms));
```

Comment:
for the batched API, can we have a tests where we have more than one Unknown reason to check that multiple different unknowns are correctly preserved in the results?

Reply:
Disregard this comment, I initially confused that with normalization API's user_meta stuff, and was wondering if we will preserve multiple (but different) Unknowns. But this is different and I just forgot to remove the comment.